### ISSUE TITLE
Update

### ISSUE THREAD

**Issue by pasha656** [bignerdranch/expandable-recycler-view#206]
I have a problem: 

When I update list I do that
```
public void update(List<?> newData)  {
    notifyParentItemRangeRemoved(0, oldData.size());
    oldData.clear();
    oldData.addAll(newData);
    notifyParentItemRangeInserted(0, newData.size());
}
```

But when I do it I have flicker. 
Can you help me and say the right way do data update ?

If I no use 
`notifyParentItemRangeRemoved(0, oldData.size());`
I did not have flicker, but I have duplication

**Comment by XxGoliathusxX** [comment]
Just clear the list first. Add the new data and call notifydatasetchanged. Giv me a feedback if it works.

**Comment by paul-turner** [comment]
@pasha656 another option is to use [swapAdapter](https://developer.android.com/reference/android/support/v7/widget/RecyclerView.html#swapAdapter(android.support.v7.widget.RecyclerView.Adapter, boolean))

The best option is currently not available but being tracked in #89
